README.md

```diff
@@ -584,6 +588,11 @@ db:
   # same as GRYPE_DB_UPDATE_URL env var
   update-url: "https://toolbox-data.anchore.io/grype/databases/listing.json"
 
+  # Max allowed age for vulnerability database,
+  # age being the time since it was built
+  # Default max age is 120h (or five days)
+  # Set it to 0 (zero) to disable check
+  max-allowed-db-age: "120h"
```

Comment:
Can we additionally allow for `d` for days for config input? I realize that go `time.ParseDuration` doesn't handle this, but it's a little awkward to configure high values.

I've got similar comments around the output that we are indicating, for example:
```
$ GRYPE_DB_AUTO_UPDATE=false grype alpine:latest
1 error occurred:
	* failed to load vulnerability db: vulnerability database is invalid (run db update to correct): the vulnerability database was built 539h0m0s ago (> max allowed 120h0m0s)
```

Something like `539h0m0s` should get converted to `22 days` (or similar) since this message is intended for the user. Consider using something like https://github.com/dustin/go-humanize#times to help with showing human readable values.

Reply:
There is no general way to parse a duration like `1d` because days don't have consistent length. More on https://github.com/golang/go/issues/11473#issuecomment-116930849

I am in favor of human friendly outputs 👍🏼

Reply:
I've added https://github.com/hako/durafmt since its implementation doesn't assume local timezone (like https://github.com/dustin/go-humanize#times does). We have to compare duration to UTC; the db built value is in UTC.